detection-rules/body_paypal_invoice_exploit_callback_phishing.yml

```diff
@@ -0,0 +1,43 @@
+name: "PayPal Invoice Abuse"
+description: |
+  A fraudulent invoice/receipt found in the body of the message sent by exploiting Paypal's invoicing service.
+  Callback Phishing is an attempt by an attacker to solicit the victim (recipient) to call a phone number. 
+  The resulting interaction could lead to a multitude of attacks ranging from Financial theft, Remote Access Trojan (RAT) Installation or Ransomware Deployment.
+type: "rule"
+references:
+  - "https://anderegg.ca/2023/02/01/a-novel-paypal-scam"
+severity: "medium"
+source: |
+   type.inbound
+   and length(attachments) == 0
+   and sender.email.domain.root_domain == "paypal.com"
+   and not any(headers.hops, .authentication_results.dmarc == "fail")
+   and strings.ilike(body.html.display_text, "*seller note*")
+   and (
+       //phone number but not 800 number
+       regex.contains(body.html.inner_text, '[\s:,-]\+?\d{1,2}[\s:,-]\(?([2-7][0-9]{2}|80[1-9]|8[1-9][0-9])\)?[\s:,-]\d{3}[\s:,-]\d{4}\b')
+       and (
+           4 of (
+               strings.ilike(body.html.inner_text, '*you did not*'),
+               strings.ilike(body.html.inner_text, '*subscription*'),
+               strings.ilike(body.html.inner_text, '*antivirus*'),
+               strings.ilike(body.html.inner_text, '*order*'),
+               strings.ilike(body.html.inner_text, '*support*'),
+               strings.ilike(body.html.inner_text, '*receipt*'),
+               strings.ilike(body.html.inner_text, '*invoice*'),
+               strings.ilike(body.html.inner_text, '*call*'),
+               strings.ilike(body.html.inner_text, '*cancel*'),
+               strings.ilike(body.html.inner_text, '*renew*'),
+               strings.ilike(body.html.inner_text, '*refund*'),
+               strings.ilike(body.html.inner_text, '*+1*'),
+               strings.ilike(body.html.inner_text, '*help*desk*')
+           )
+       )
+       or (
```

Comment:
this `or` is at the same level as the previous `and`, so really i think we're or'ing these 3 expressions. i'm fine with it if you are, just making sure that's what you intended!

Reply:
Good call that or should be matching the upper and. I think it was functioning appropriately as the parens were correct but yeah it was indented incorrectly.